src/core/Microsoft.PowerToys.Settings.UI.Lib/Interface/IGeneralSettingsData.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.PowerToys.Settings.UI.Lib.Interface
+{
+    public interface IGeneralSettingsData
```

Comment:
@alekhyareddy28 Why is this an interface?

Reply:
Presently, General settings is being passed to outGoingGeneralSettings.cs as an argument. I thought it would be better for testing purposes to pass it as an interface instead.
Other than that, I don't have a strong reason to keep this interface, we can remove it if you think it's not needed. Then I can pass `GeneralSettings` directly in the `OutGoingGeneralSettings` constructor instead of `IGeneralSettingsData`.